pkg/codegen/internal/test/testdata/simple-enum-schema/go/plant/pulumiEnums.go

```diff
@@ -0,0 +1,101 @@
+// *** WARNING: this file was generated by test. ***
+// *** Do not edit by hand unless you're certain you know what you are doing! ***
+
+package plant
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+
+	"github.com/pulumi/pulumi/sdk/v2/go/pulumi"
+)
+
+// plant container colors
+type ContainerColor pulumi.String
+
+const (
+	Red    = ContainerColor("red")
```

Comment:
Good catch. I updated to use <typename><enum val> instead since it seems a bit more idiomatic and easier to discover options through IDE intellisense. I also added a change to use a `_` separator if the enum value contains `_`. See https://github.com/pulumi/pulumi/pull/5670/files#diff-0b3967193235a3bfca3614bcccb84337da74174690c263aecdcc2216c42239d2R17 for an example.